chrome-finder.ts

```diff
@@ -229,10 +229,17 @@ function findChromeExecutables(folder: string): Array<string> {
     // Output of the grep & print looks like:
     //    /opt/google/chrome/google-chrome --profile-directory
     //    /home/user/Downloads/chrome-linux/chrome-wrapper %U
-    let execPaths = execSync(`grep -ER "${chromeExecRegex}" ${folder} | awk -F '=' '{print $2}'`)
-                        .toString()
-                        .split(newLineRegex)
-                        .map((execPath: string) => execPath.replace(argumentsRegex, '$1'));
+    let execPaths;
+
+    try {
+      execPaths = execSync(`grep -ER "${chromeExecRegex}" ${folder} | awk -F '=' '{print $2}'`);
```

Comment:
want to add a quick comment about some platforms not supporting `-R`, maybe with a link to the issue?